tests/test_infer/test_kvcache_manager.py

```diff
@@ -147,6 +149,16 @@ def test_cache_manager(test_config):
     assert cache_manager.get_num_available_blocks() == num_blocks
 
 
+def run_dist(rank, world_size, port):
+    colossalai.launch(config={}, rank=rank, world_size=world_size, port=port, host="localhost")
+    check_cache_manager()
+
+
+@pytest.mark.dist
+def test_cache_manager():
+    spawn(run_dist, 1)
+
+
```

Comment:
Why we launch the distributed environment here? Based on the lines changed, the cache manager itself is not supposed to be tested on a distributed env, and the code changes in this test file seem to be unnecessary and redundant.

Reply:
I added a statement to config.py to determine the validity of pp_size and tp_size, which uses torch.dist.